tests/e2e/fetchUserContacts.test.ts

```diff
@@ -35,7 +38,7 @@ describe('web3mail.fetchMyContacts()', () => {
       data: { email: 'test2@gmail.com' },
       name: 'test do not use',
     });
-  }, 3 * MAX_EXPECTED_BLOCKTIME);
+  }, 2 * MAX_EXPECTED_BLOCKTIME + MAX_EXPECTED_WEB2_SERVICES_TIME);
```

Comment:
2 tx => 2 * (1 block to submit tx + 1 block for confirmation) => 4
```suggestion
  }, 4 * MAX_EXPECTED_BLOCKTIME + MAX_EXPECTED_WEB2_SERVICES_TIME);
```